app/lib/actions/katello/actions_jail.rb

```diff
@@ -0,0 +1,40 @@
+module Actions
+  module Katello
+    module ActionsJail
+      def content_view_id
+        input['content_view']['id']
+      end
+
+      def content_view_name
+        input['content_view']['name']
+      end
+
+      def content_view_label
+        input['content_view']['label']
+      end
+
+      def organization_id
+        input['organization']['id']
+      end
+
+      def organization_name
+        input['organization']['name']
+      end
+
+      def organization_label
+        input['organization']['label']
+      end
+
+      def self.included(base)
+        super
+        base.instance_eval do
+          jail_class = Class.new(::Actions::ObservableAction::Jail) do
+            allow :content_view_id, :content_view_name, :content_view_label,
+                  :organization_id, :organization_name, :organization_label
+          end
+          base.const_set('Jail', jail_class)
+        end
+      end
```

Comment:
I am just wondering if it is not cleaner just to define attributes in this module and simply repeating all allowed attributes for each class. But I am okay with this.

Reply:
Sure it would be cleaner than some magic Ruby-woodoo, but repeating is not so good as well since one might forget to update allowed methods somewhere else.